src/repository/repository.action.js

```diff
@@ -140,6 +142,28 @@ export const getIssues = url => {
   };
 };
 
+export const checkReadMe = url => {
+  return (dispatch, getState) => {
+    const accessToken = getState().auth.accessToken;
+
+    dispatch({ type: GET_REPO_README_STATUS.PENDING });
+
+    fetchUrlHead(url, accessToken)
+      .then(data => {
+        dispatch({
+          type: GET_REPO_README_STATUS.SUCCESS,
+          payload: !(data.status === 404),
```

Comment:
`data.status !== 404` seems clearer to me

Reply:
Ha, I agree with you and should apology for copying a bad example nearby. Can I refactor all similar usages in this pr?

Reply:
No need to apologize :)
I think it's better to fix other cases in another PR, so that we keep your PR focused on the Readme stuff

Reply:
Updated. I will send another PR to fix other cases after this one has been merged.